draft-ietf-webtrans-http2.md

```diff
@@ -251,368 +285,353 @@ unidirectional, indicated by the second least significant bit of the
 stream ID.
 
 Because WebTransport does not provide an acknowledgement mechanism for
-WebTransport frames, it relies on the underlying transport's in order delivery
+WebTransport capsules, it relies on the underlying transport's in order delivery
 to inform stream state transitions. Wherever QUIC relies on receiving an ack
 for a packet to transition between stream states, WebTransport performs that
 transition immediately.
 
-# WebTransport Frames
+# WebTransport Capsules
 
-WebTransport frames mirror their QUIC counterparts as closely as possible to
-enable maximal reuse of any applicable QUIC infrastructure by implementors.
+WebTransport capsules mirror their QUIC frame counterparts as closely as
+possible to enable maximal reuse of any applicable QUIC infrastructure by
+implementors.
 
-A WebTransport frame begins with a Frame Type and Frame Length which are
-followed by zero or more fields that are type-dependent.
+WebTransport capsules follow the Capsule Protocol format defined in {
```

Comment:
I would just say "WebTransport uses the capsule protocol" with a reference instead of repeating the format.

Reply:
Great plan, done